tests/test_triso/inputs_true.dat

```diff
@@ -1 +1 @@
-f33e6653b883200457df2ff2ba9cf715d5ddaa1296dd71d277c6f1d9d5b7831cc92aaf1e97509d26e5a93235cd9f775c0cfaa5ebc3dfe8fc71469bac166d362b
\ No newline at end of file
+2285ba99573743929cee590e2ba4d86becbf38d58af765498b73425f2fa3ccc3e0d20a59260d283a3ff39038d87e12142e0156b22152ccecbe2609a291d1d347
```

Comment:
I think you need to run `python test_triso.py --update` once more so that the inputs hash gets updated. When you updated the nuclide names in the test, it caused changes in the XML files and hence the hash here.